src/ds/test/hash_bench.cpp

```diff
@@ -26,7 +26,11 @@ static void hash(picobench::state& s)
   }
 }
 
-const std::vector<int> hash_sizes = {1, 8, 64, 1024, 16536};
+#ifdef SAN
```

Comment:
There probably is something I'm missing but what is the value in running micro-benchmarks with SAN on?